website_sale_product_minimal_price/static/src/js/website_sale_product_minimal_price.js

```diff
@@ -0,0 +1,80 @@
+odoo.define("website_sale_product_minimal_price.load", function (require) {
+    "use strict";
+    var ajax = require("web.ajax");
+    var core = require("web.core");
+    var field_utils = require("web.field_utils");
+    var ProductConfiguratorMixin = require(
+        "website_sale_stock.ProductConfiguratorMixin");
+    var QWeb = core.qweb;
+    var load_xml = ajax.loadXML(
+        "/website_sale_product_minimal_price/static/src/xml/" +
+            "website_sale_product_minimal_price.xml",
+        QWeb
+    );
+
+    // Save original method
+    var _onChangeCombinationStock =
+        ProductConfiguratorMixin._onChangeCombinationStock;
+    ProductConfiguratorMixin._onChangeCombinationStock = function (
+        ev,
+        $parent,
+        combination
+    ) {
+        var self = this;
+        var args = arguments;
+        if (!this.isWebsite) {
+            return;
+        }
+        ajax.jsonRpc("/sale/get_combination_info_pricelist_atributes", "call", {
+            product_id: combination.product_id,
+            actual_qty: $(".quantity").val(),
+        }).then(function (unit_prices) {
+            $(".temporal").remove();
+            if (unit_prices.length > 0) {
+                load_xml.then(function () {
+                    var $form = $('form[action="/shop/cart/update"]');
+                    $form.append('<hr class="temporal"/>');
+                    $form.append(
+                        QWeb.render("website_sale_product_minimal_price.head", {
+                            uom: combination.uom_name,
+                            symbol: unit_prices[0].currency.symbol,
+                        })
+                    );
+                    var $first_row = $("#first_row");
+                    var $second_row = $("#second_row");
+                    for (var i in unit_prices) {
+                        if (!unit_prices[i].price) {
```

Comment:
This means "if not exists price" or "if price is 0"?

Reply:
If price is 0, I change it to unit_prices[i].price === 0